public/docs/_examples/server-communication/dart/web/main.dart

```diff
@@ -0,0 +1,25 @@
+// #docregion
+import 'package:angular2/angular2.dart';
+import 'package:angular2/bootstrap.dart';
+import 'package:http/browser_client.dart';
+import 'package:http_in_memory_web_api/http_in_memory_web_api.dart';
+import 'package:server_communication/toh/toh_component.dart';
+import 'package:server_communication/hero_data.dart';
+import 'package:server_communication/wiki/wiki_component.dart';
+import 'package:server_communication/wiki/wiki_smart_component.dart';
+
+main() {
+  bootstrap(TohComponent, [
+//#enddocregion
+//#docregion in-mem-web-api-providers
+// in-memory web api providers
+    provide(BrowserClient,
+        useValue:
+            new HttpClientInMemoryBackendService(heroData)), // in-mem server
```

Comment:
I remember I had some trouble with the annotation requiring a const class or something. I'll take a look into it again.